Sources/MessagingInApp/Views/GistInlineInAppMessageView.swift

```diff
@@ -100,7 +100,7 @@ public class GistInlineMessageUIView: UIView {
     }
 
     deinit {
-        unsubscribeFromInAppMessageState()
+        dismissCurrentMessage()
```

Comment:
thinking if it would work for UI kit as well?

Reply:
Not entirely sure at the moment. I'm currently working on UIKit and will verify and fix this in upcoming PR if needed.